src/+io/readGroundwaterSettings.m

```diff
@@ -0,0 +1,24 @@
+function GroundwaterSettings = readGroundwaterSettings()
```

Comment:
```suggestion
function GroundwaterSettings = readGroundwaterSettings()
```
Shouldn't this be "initializeGroundwaterCoupling"? No settings are loaded here.

Reply:
Can we wait for the final name of the function till all my implementations are finished😀? the 'readGroundwaterSettings' function will most likely be expanded with potential settings